Packs/CortexXDR/Integrations/XDR_iocs/XDR_iocs.py

```diff
@@ -431,9 +457,10 @@ def iocs_command(client: Client):
         path, iocs = prepare_enable_iocs(indicators)
     else:  # command == 'disable'
         path, iocs = prepare_disable_iocs(indicators)
+    demisto.info(f"IOCs command: sending {len(iocs)} IOCs to endpoint {path}")
     requests_kwargs: dict = get_requests_kwargs(_json=iocs)
     client.http_request(url_suffix=path, requests_kwargs=requests_kwargs)
-    return_outputs(f'indicators {indicators} {command}d.')
+    return_outputs(f'IOCs command: {command}d {indicators=}')
```

Comment:
```suggestion
    return_outputs(f'IOCs command: {command} {indicators=}')
```

Reply:
intentional: `delete` -> `deleted`, same for `create`